cloudify_ansible/utils.py

```diff
@@ -318,14 +345,29 @@ def handle_source_from_string(filepath, _ctx, new_inventory_path):
     return new_inventory_path
 
 
+def create_ansible_cfg(ctx):
+    workspace_dir = get_instance(ctx).runtime_properties.get(WORKSPACE)
+    collections_location = _get_collections_location(ctx)
+    ansible_cfg_file = os.path.join(workspace_dir, 'ansible.cfg')
+    with open(ansible_cfg_file, 'w') as f:
+        f.write("[defaults]\n")
+        f.write("collections_path={}\n".format(collections_location))
+        f.write("roles={}\n".format(workspace_dir))
+
+
 def create_playbook_workspace(ctx=None):
     """ Create a temporary folder, so that we don't overwrite fields.
-
     :param ctx: The Cloudify context.
     :return:
     """
     get_instance(ctx).runtime_properties[WORKSPACE] = \
         mkdtemp(dir=get_node_instance_dir())
+    create_ansible_cfg(ctx)
+
+
+def delete_temp_folder(directory):
+    if directory and os.path.exists(directory):
+        shutil.rmtree(directory)
 
 
```

Comment:
Should `delete_playbook_workspace` be deleted during install install if it fails?